cmd/config/cache/lookup.go

```diff
@@ -142,15 +142,15 @@ func LookupConfig(kvs config.KVS) (Config, error) {
 		cfg.MaxUse = cfg.Quota
 	}
 
-	if triggerhitsStr := env.Get(EnvCacheTriggerHits, kvs.Get(TriggerHits)); triggerhitsStr != "" {
-		cfg.TriggerHits, err = strconv.Atoi(triggerhitsStr)
+	if afterStr := env.Get(EnvCacheAfter, kvs.Get(After)); afterStr != "" {
+		cfg.After, err = strconv.Atoi(afterStr)
 		if err != nil {
-			return cfg, config.ErrInvalidCacheTriggerHits(err)
+			return cfg, config.ErrInvalidCacheAfter(err)
 		}
-		// triggerHits should be a valid value >= 0.
-		if cfg.TriggerHits < 0 {
-			err := errors.New("config trigger hits value should not be less than 0")
-			return cfg, config.ErrInvalidCacheTriggerHits(err)
+		// after should be a valid value >= 0.
+		if cfg.After < 0 {
+			err := errors.New("config after value cannot be less than 0")
+			return cfg, config.ErrInvalidCacheAfter(err)
```

Comment:
The message doesn't make sense `config after value` has no meaning.

Reply:
It would be better to call it `cache after value`

Reply:
fixed